source/infrastructure/dev-tools/index.html.md.erb

```diff
@@ -91,3 +91,26 @@ Run `rvm install ruby 2.7.4`
 Run `rvm use 2.7.4`
 9. install yarn
 Run `brew install yarn`
+
+## Asdf
+
+### Introduction
+
+[Asdf](https://asdf-vm.com) is a version manager used for manging software you install on your computer, it’s a command line interface
+tool that handles installing multiple versions of software packages, allowing you to switch between those versions at runtime.
+
+Each software package is managed by a plugin, an official list of plugins can be found [here](https://github.com/asdf-vm/asdf-plugins).
```

Comment:
Could we also mention one major advantage of asdf is that since the tool-versions file is committed to the repo, it ensures consistency amond the developers?